app/Http/Controllers/Store/CentreController.php

```diff
@@ -50,20 +52,51 @@ public function printCentreCollectionForm(Centre $centre)
     }
 
     /**
-     * Exports a summary of registrations from the User's relevant Centres.
+     * Exports a summary of registrations from the User's relevant Centres or specified Centre.
      *
-     * @return \Illuminate\Contracts\Routing\ResponseFactory|\Symfony\Component\HttpFoundation\Response
+     * @param Centre $centre
+     * @return ResponseFactory|\Symfony\Component\HttpFoundation\Response
      */
-    public function exportRegistrationsSummary()
+    public function exportRegistrationsSummary(Centre $centre)
     {
         // Get User
         $user = Auth::user();
 
-        // Get now()
-        $now = Carbon::now();
+        if (isset($centre->id)) {
+            // Set for specified centre
+            $centre_ids = [$centre->id];
+            $dateFormats = [
+                'dob' => 'm/Y'
+            ];
+        } else {
+            // Set for relevant centres
+            $centre_ids = $user->relevantCentres()->pluck('id')->all();
+            $dateFormats = [];
+        }
 
-        // Get centre ids
-        $centre_ids = $user->relevantCentres()->pluck('id')->all();
+        $summary = $this->getCentreRegistrationsSummary($centre_ids, $dateFormats);
+
+        return $this->streamFile(
+            $this->writeExcelDoc($summary)
+        );
+    }
+
+    /**
+     * Returns array of formatted data about the Centre's Registrations
+     *
+     * @param array $centre_ids
+     * @param array $dateFormats
+     * @return array
+     */
+    private function getCentreRegistrationsSummary(array $centre_ids, $dateFormats = [])
```

Comment:
Does anything else use this function? Maybe you don't need the `$dateFormats` param?

Reply:
The decision about whihc formats is done in the controller function. I was considering moving this to registration, one reason i pulled it out into a function, the other being purpose separation.